matching.go

```diff
@@ -206,6 +207,10 @@ func ReducePeople(people People, matcher external.Matcher, blacklist Blacklist,
 		}
 	}
 	mean, std := stat.MeanStdDev(componentsSize, nil)
+	// If std is NaN, serialization to JSON will fail
+	if math.IsNaN(std) {
+		std = 0
```

Comment:
LOL, this is the fix I had to do independently in https://github.com/src-d/identity-matching/issues/69